cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,21 +1,24 @@
+import {SauceContentHomePage, SauceStandardLogin, SauceInventory, AddProductToCart, SauceCart, SauceCheckoutInformation,
+  CheckoutOverview, FinishCheckout} from "../page/index";
+
+const menuContentPage = new SauceContentHomePage();
+const LoginPage = new SauceStandardLogin();
+const InventoryPage = new SauceInventory();
+const InventoryDetailsPage = new AddProductToCart();
+const ShoppingCartPage = new SauceCart();
+const CheckoutInformationPage = new SauceCheckoutInformation();
+const CheckoutOverviewPage = new CheckoutOverview();
+const CheckoutCompletePage = new FinishCheckout();
```

Comment:
variables should be named in camelCase format. So instead of LoginPage you should use loginPage, and so on..

Reply:
Fixed in the latest commit, please check @holgiosalos @JuanDelahozMiranda